test/tensors/tensors.jl

```diff
@@ -269,28 +271,27 @@ for V in spacelist
             end
         end
         @timedtestset "Full trace: test self-consistency" begin
-            t = rand(ComplexF64, V1 ⊗ V2' ⊗ V2 ⊗ V1')
-            t2 = permute(t, ((1, 2), (4, 3)))
-            s = @constinferred tr(t2)
-            @test conj(s) ≈ tr(t2')
+            t = rand(ComplexF64, V1 ⊗ V2' ← V1 ⊗ V2')
+            s = @constinferred tr(t)
+            @test conj(s) ≈ tr(t')
             if !isdual(V1)
-                t2 = twist!(t2, 1)
+                t2 = twist!(t, 1)
             end
             if isdual(V2)
-                t2 = twist!(t2, 2)
+                t2 = twist!(t, 2)
             end
             ss = tr(t2)
-            @tensor s2 = t[a, b, b, a]
-            @tensor t3[a, b] := t[a, c, c, b]
-            @tensor s3 = t3[a, a]
+            @plansor s2 = t[a b; a b]
+            @plansor t3[a; b] := t[a c; b c]
+            @plansor s3 = t3[a; a]
```

Comment:
I don't think you can simply change this to `@plansor` without affecting the results, so I think it might be better to just skip these tests if the braidingstyle doesn't allow it?

Reply:
I'm actually really confused `@tensor` and `@planar` give the same result to begin with, and looking at it a bit more in detail I think `tr` usually should correspond to `@planar`, so I don't know why `@tensor` was working before...